### ISSUE TITLE
yarn install fails with `401 Unauthorized` error

### ISSUE THREAD

**Issue by zaibon** [threefoldtech/tf_cockpit#9]
```
▶ yarn install
yarn install v1.17.3
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
[2/4] Fetching packages...
error An unexpected error occurred: "https://npm.fontawesome.com/@fortawesome/fontawesome-pro/-/fontawesome-pro-5.10.0.tgz: Request failed \"401 Unauthorized\"".
info If you think this is a bug, please open a bug report with the information provided in "/home/zaibon/dev/go/src/github.com/threefoldtech/tf_cockpit/frontend/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by zaibon** [comment]
https://github.com/FortAwesome/Font-Awesome/issues/12609

**Comment by coeneivan** [comment]
Fixed with 580ecadec5fe123e72fdfafc50b2c6b3964016ee
